internal/queue/background.go

```diff
@@ -84,9 +84,13 @@ func New(ctx context.Context) *Queue {
 
 func (q *Queue) run(ctx context.Context) {
 	for t := range q.work {
-		if err := t(ctx); err != nil {
+		ctx2,err := t(ctx)
```

Comment:
These files don't seem to be properly formatted.

Reply:
I believe this is now fixed, but I'll let you be the judge of that